tests/message_streaming_decoder_test.c

```diff
@@ -282,15 +298,15 @@ static int s_test_streaming_decoder_incoming_application_one_int32_header_pair_v
     (void)ctx;
     /* clang-format off */
     uint8_t test_data[] = {
-        0x00, 0x00, 0x00, 0x1e,         /* total length */
-        0x00, 0x00, 0x00, 0x0e,         /* headers length */
-        0x5d, 0x4a, 0xdb, 0x8d,         /* prelude crc */
+        0x00, 0x00, 0x00, 0x1b,         /* total length */
+        0x00, 0x00, 0x00, 0x0b,         /* headers length */
+        0xe5, 0xc0, 0xa0, 0x72,         /* prelude crc */
```

Comment:
just curious, why are these changing?
was this data bad, and the new callback confirmed it?

Reply:
Yes